milatools/cli/commands.py

```diff
@@ -801,3 +727,193 @@ def _forward(
+        ssh_config,
+        "mila-cpu",
+        User=username,
+        Port=2222,
+        ForwardAgent="yes",
+        StrictHostKeyChecking="no",
+        LogLevel="ERROR",
+        UserKnownHostsFile="/dev/null",
+        RequestTTY="force",
+        ConnectTimeout=600,
+        ProxyCommand=(
+            """ssh mila "salloc --partition=unkillable --dependency=singleton """
+            """--cpus-per-task=2 --mem=16G """
+            '''/usr/bin/env bash -c 'nc \\$SLURM_NODELIST 22'"'''
+        ),
+        RemoteCommand="srun --cpus-per-task=2 --mem=16G --pty /usr/bin/env bash -l",
+        _input=_input,
+    ):
+        changed_entries_in_config.append("mila-cpu")
+
+    if _add_ssh_entry_interactive(
+        ssh_config,
+        "mila-gpu",
+        User=username,
+        Port=2222,
+        ForwardAgent="yes",
+        StrictHostKeyChecking="no",
+        LogLevel="ERROR",
+        UserKnownHostsFile="/dev/null",
+        RequestTTY="force",
+        ConnectTimeout=600,
+        ProxyCommand=(
+            """ssh mila "salloc --partition=unkillable """
+            """--dependency=singleton --cpus-per-task=2 --mem=16G --gres=gpu:1 """
+            '''/usr/bin/env bash -c 'nc \\$SLURM_NODELIST 22'"'''
+        ),
+        RemoteCommand=(
+            "srun --cpus-per-task=2 --mem=16G --gres=gpu:1 --pty /usr/bin/env bash -l",
+        ),
+        _input=_input,
+    ):
+        changed_entries_in_config.append("mila-gpu")
+
+    # Check for *.server.mila.quebec in ssh config, to connect to compute nodes
+
+    cnode_pattern = "*.server.mila.quebec !*login.server.mila.quebec"
+
+    # TODO: What if there is a cnode_pattern and a '*.server.mila.quebec' entry?
+    if "*.server.mila.quebec" in ssh_config.hosts():
+        if yn(
+            "The '*.server.mila.quebec' entry in ~/.ssh/config is too general and should "
+            "exclude login.server.mila.quebec. Fix this?",
+            _input=_input,
+        ):
+            ssh_config.rename("*.server.mila.quebec", cnode_pattern)
+            changed_entries_in_config.append(cnode_pattern)
+    elif _add_ssh_entry_interactive(
+        ssh_config,
+        cnode_pattern,
+        HostName="%h",
+        User=username,
+        ProxyJump="mila",
+        _host_name_for_prompt="*.server.mila.quebec",
+        _input=_input,
+    ):
+        changed_entries_in_config.append(cnode_pattern)
+
+    # TODO: Might be better to display a diff of the current and the potential new config instead.
+    if not changed_entries_in_config:
+        print("Did not change ssh config")
+    elif not _confirm_changes(ssh_config, hosts=changed_entries_in_config, _input=_input):
+        exit("Did not change ssh config")
+    else:
+        ssh_config.save()
+        print(f"Wrote {ssh_config_path}")
+
+
+def _confirm_changes(
+    ssh_config: SSHConfig, hosts: list[str], _input: Optional[PipeInput] = None
+) -> bool:
+    print(T.bold("The following code will be appended to your ~/.ssh/config:\n"))
```

Comment:
```suggestion
    print(T.bold("The following code will be appended or modified in your ~/.ssh/config:\n"))
```

since there is potentially a rename on `*.server.mila.quebec`?

Reply:
Added this change (not sure in which commit exactly)